### ISSUE TITLE
Add CLI option to override schedule.

### ISSUE THREAD

**Issue by mastermatt** [renovateapp/renovate#1731]
#### This is a:

* [ ] Bug report (non-security related)
* [x] Feature request
* [ ] I'm not sure which of those it is

#### I'm using:

* [ ] The Renovate GitHub App
* [x] Self-hosted GitHub
* [ ] Self-hosted GitLab
* [ ] Self-hosted VSTS

#### Please describe the issue:
I'd like the ability to manually run Renovate on a repo with the CLI and have it process regardless of the schedule settings in the repo.
By the looks of `renovate --help` my only option at the moment is to remove the schedule config, commit/push, run Renovate, then add the config back.

This new option could take the form of either a flag to simply ignore the schedule or a cli option to override the schedule config. The latter may be a can of worms as schedules can also be set per package and I'm not sure if it adds value anyway.

**Comment by rarkins** [comment]
Yes, I would find this useful for the app as well. I'm thinking maybe an admin (bot owner) flag called `ignoreSchedules` that ignores all schedules.

**Comment by rarkins** [comment]
Here's another idea. What about a CLI option that itself means "CLI overrides repository", e.g. might be like

```
renovate foo/bar --schedule="[]" --cli-overrides
```

Cc @hulkish

**Comment by mastermatt** [comment]
As a user, I would always expect/hope that the cli would accept the same general config options and override the flat file configs. Or at the very least an option to provide a JSON string or a JSON file path that would get merged on top of the flat file config[s].

That being said, that seems like a larger ask than my specific needs currently so... 🤷‍♂️

**Comment by rarkins** [comment]
Idea: add a new `force` object that gets applied last. And then add a `forceCli` option that copies any cli config into force, therefore forcing cli options

**Comment by rarkins** [comment]
@mastermatt you should now be able to do it something like this to remove any schedules in a repository you're running on:

`renovate --schedule= --force-cli user/repo`

See https://github.com/renovateapp/renovate/pull/2145 for details
